pages/common/aws-help.md

```diff
@@ -1,7 +1,7 @@
 # aws help
 
 > Display help information about the AWS CLI.
-> More information: <https://docs.aws.amazon.com/cli/latest/reference/>.
+> More information: <https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-help.html>.
```

Comment:
In my opinion pointing towards reference makes more sense.

Reply:
The original link points to the generic `aws cli` reference:
https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-help.html

But seeing this is specific for the `aws help` command, so I thought the link:
https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-help.html

is more relevant, as the page proves further details on how to use `aws help`.

Reply:
I agree with the change. The `help` subcommand isn't even listed in the old link.